decider-sequenza/src/main/java/ca/on/oicr/pde/deciders/SequenzaDecider.java

```diff
@@ -102,19 +134,78 @@ protected ReturnValue doFinalCheck(String commaSeparatedFilePaths, String commaS
                 if (!bs.getPath().equals(p)) {
                     continue;
                 }
-                String tt = bs.getTissueType();
+                String tt = bs.tissueType;
+                if (!tt.isEmpty() && !tt.equals("R")){
```

Comment:
I would have put this into a statis variable instead of hard-coding it as plain string "R"